client/src/pages/sessions/index.tsx

```diff
@@ -276,15 +276,19 @@ function SessionsTable(props: {
   }, [context.onlyCreator, context.showGraded]);
 
   React.useEffect(() => {
-    const filter: any = {};
+    const filter: {
+      lessonCreatedBy?: string;
+      lessonId?: string;
+      // graderGrade?: string | null;
+    } = {};
     if (context.onlyCreator) {
-      filter["lessonCreatedBy"] = context.user?.name;
-    }
-    if (!context.showGraded) {
-      filter["graderGrade"] = null;
+      filter.lessonCreatedBy = context.user?.name;
     }
+    // if (!context.showGraded) {
```

Comment:
Are we getting rid of filtering by grader grade? Should remove these comments?

Reply:
let me double check, but I thought I was commenting out because it was never used

Reply:
does something actually use that `showGraded` value? If you look at the diff, you'll see it always passes a `null` value to gql